spec/controllers/questions/mailing_address_controller_spec.rb

```diff
@@ -80,7 +80,7 @@
       end
 
       it "enqueues a job to generate F13614cPdf" do
-        expect(GenerateF13614cPdfJob).to receive(:perform_later).with(intake.client.id, "Preliminary 13614-C.pdf")
+        expect(GenerateF13614cPdfJob).to receive(:perform_later).with(intake.id, "Preliminary 13614-C.pdf")
```

Comment:
parity with the code -- i think sometimes client.id == intake.id, but not always